miscellaneous/WoeUSB-ng.desktop

```diff
@@ -1,8 +1,8 @@
-#!/usr/bin/env xdg-open
 [Desktop Entry]
+Version=1.0
 Name=WoeUSB-ng
 Exec=woeusbgui
-Icon=/usr/share/icons/WoeUSB-ng/icon.ico
+Icon=woeusb-logo
```

Comment:
* The `-logo` suffix is redundant as files under /usr/share/icons directory should have the same meaning
* Use `woeusb-ng` instead.
* setup.py should also be fixed: <https://github.com/WoeUSB/WoeUSB-ng/blob/bcb75832cabb02fea4f0f8311366051caed7160f/setup.py#L26> for the icon to become effective.

Reply:
No need to worry. The `data_files` option takes care of it. `icon.ico` is accessed only by the source code and not the assets.